---
layout: page-fullwidth
title: "Speakers & Presentations"
header: no
permalink: /speakers/
---

<div id="speaker1" style="display: flex; flex-direction: column;">

<h3>Scaling Insights in Cancer Care with Machine Learning</h3>

<h4>Will Shapiro, Data Science and Machine Learning at Flatiron Health</h4>

<p>
<img class="imgfloat" src="../docs/Shapiro.jpg"/>

Will Shapiro currently leads Data Insights Engineering at Flatiron, which encompasses data science, machine learning, artificial intelligence and product analytics. He became fascinated by machine learning while building personalization engines at Spotify, where he led AI teams focused on foundational research. He is a prolific inventor, with 11 (and counting) granted patents in AI, ML and personalization, as well as an author of a pivotal study that used listening behaviors to predict personality type. After experiencing firsthand the profound difference between biomarker-targeted cancer therapy and traditional chemotherapy, Will became passionate about the potential of personalized medicine - and in particular - ensuring that the future of medicine is personalized for everyone, not just the targets of traditional clinical trials.
</p>


<h4>Abstract</h4>

TBD

</div>

----

<div id="speaker2" style="display: flex; flex-direction: column;">

<h3>Advancing Clinical Development through Real-World Data, AI and Machine Learning</h3>

<h4>Harry Yang, Vice President of Biometrics at Recursion Pharmaceuticals</h4>

<p>
<img class="imgfloat" src="../docs/Harry.png"/>

Harry Yang, Ph.D., is Vice President of Biometrics at Recursion Pharmaceuticals, following
positions as VP of Biometrics at Fate Therapeutics and Head of Statistical Science at
AstraZeneca and MedImmune. With 26 years of experience in small molecule, biologics, and
cellular immunotherapy development, his expertise spans the therapeutics areas of
transplantation, vaccine, autoimmune & inflammatory disease, oncology, and rare disease. Dr.
Yang is well-versed in innovative trial design, regulatory submissions, real-world data utilization,
and the integration of AI and machine learning in drug R&D. He is a prolific author, having
published 8 books and over 130 articles and book chapters covering diverse statistical,
scientific, and regulatory topics in drug R&D. Additionally, he serves as the Vice Chair of the USP
Statistics Expert Committee.
</p>


<h4>Abstract</h4>

In the ever-evolving landscape of biomedical research, the confluence of recent
breakthroughs in real-world data (RWD), artificial intelligence (AI), and machine learning has
paved the way for unlocking the untapped potential residing within the vast reservoirs of
historical and ongoing clinical trial data. This presentation delves into the transformative power

of these advancements, showcasing two use cases that illuminate the pivotal role of RWD and
machine learning models in shaping and enhancing clinical development strategies, specifically
within the area of immune-oncology products.

</div>

----

<div id="speaker3" style="display: flex; flex-direction: column;">

<h3>Active and Federated Learning in Drug Discovery and Development</h3>

<h4>Christopher James Langmead, Scientific Executive Director at Amgen</h4>

<p>
<img class="imgfloat" src="../docs/Chris.png"/>

Dr. Langmead leads Amgen's efforts in the development and application of AI/ML-based methods for the discovery and optimization of biologics. His team at Amgen is involved with all stages of the pipeline. Prior to joining Amgen, Dr. Langmead was a tenured faculty member in the School of Computer Science at Carnegie Mellon University where his research concerned the development of Generative AI methods for the design of proteins, and algorithms for automatic scientific discovery and sequential optimization.
</p>


<h4>Abstract</h4>

This presentation will discuss Amgen's use of Active and Federated Learning to enhance its Generative Biology platform for engineering biologics.

</div>

----

<div id="speaker5" style="display: flex; flex-direction: column;">

<h3>Multimodal Integration in the Age of Million Cells and Billion Parameters</h3>

<h4>Himel Mallick, Assistant Professor in Department of Population Health Sciences, Weill Cornell Medicine, Cornell University</h4>

<p>
<img class="imgfloat" src="../docs/Himel.jpg"/>

Himel Mallick is a data scientist and computational biologist with almost two decades of experience in Statistics, Biostatistics, and AI/ML in academia and industry. His methodological research interests are in Bayesian statistics, machine learning, and omics data science methods including multi-omics integration, microbiome, single-cell, spatial transcriptomics, imaging, and digital pathology. He has a highly cited publication track record with over 40+ publications in top-tier scientific journals including Nature and Lancet as well as top health science journals such as Statistics in Medicine and PLoS Computational Biology. He is the lead developer of several popular Bioconductor packages including MaAsLin2. He is a recipient of the IISA Early Career Award in Statistics in Data Sciences (ECASDS), a Fellow of the American Statistical Association (FASA) and an Elected Member of the International Statistical Institute.
</p>


<h4>Abstract</h4>

Research in machine learning and data science is increasingly entering the realm of staggeringly large multiview data collections (concurrent measurements (views) collected on the same subjects from multiple sources). Fueled by an explosion in recent high-throughput and AI technologies, we are now ready to enter the world of personalized medicine and individualized solutions, where clinical or other non-therapeutic interventions can be custom-tailored to individuals to achieve better outcomes based on their multiview profiles. Although analyses of such multimodal datasets have the potential to provide new insights into the underlying mechanistic processes that cannot be inferred with a single modality, the integration of very large, complex, multimodal data represents a considerable statistical and computational challenge. An understanding of the principles of data integration and visualization methods is thus necessary to determine which methods are best applied to a particular integration problem. In this talk, I will discuss open challenges in multimodal integration, including methodological issues that must be resolved to establish the resources needed to move beyond incremental advances toward translational intervention while keeping machine learning and data science at the forefront of the next generation of multiview research.

</div>


----

<div id="speaker6" style="display: flex; flex-direction: column;">

<h3>Mapping the Mind: Modeling Brain Connectivity and Its Link to Behavior</h3>

<h4>Yize Zhao, Associate Professor in the Department of Biostatistics, Yale School of Public Health, Yale University</h4>

<p>
<img class="imgfloat" src="../docs/Yize.jpg"/>

Dr. Zhao is an Associate Professor in the Department of Biostatistics at Yale School of Public Health. She
is also affiliated with Yale Center for Analytical Sciences, Yale Alzheimer&#39;s Disease Research Center, Yale
Wu Tsai Institute, Yale Center for Brain and Mind Health and Yale Computational Biology and
Bioinformatics. Her main research focuses on the development of statistical and machine learning
methods to analyze large-scale complex data (imaging, -omics, EHRs), Bayesian methods, feature
selection, predictive modeling, data integration, missing data and network analysis. She has strong
interests in biomedical research areas including mental health, mental disorders and aging, etc.
Her most recent research agenda includes analytical method development and applications on brain
network analyses, multimodal imaging modeling, imaging genetics, and the integration of biomedical
data with EHR data. Her research is supported by multiple NIH grants.
</p>


<h4>Abstract</h4>

Brain functional connectivity or connectome, a unique measure for brain functional organization,
provides a great potential to explain the neurobiological underpinning of behavioral profiles. In the first
project, we study the complex impact of multi-state functional connectivity on behaviors by analyzing
the data from a recent landmark brain development and child health study. We propose a
nonparametric, Bayesian supervised heterogeneity analysis to uncover neurodevelopmental subtypes
with distinct effect mechanisms. We impose stochastic block structures to identify network-based
functional phenotypes and develop a variational expectation–maximization algorithm to facilitate an
efficient posterior computation. Through integrating resting-state and task-related functional
connectomes, we dissect heterogeneous effect mechanisms on children’s fluid intelligence from the
functional network phenotypes including Fronto-parietal Network and Default Mode Network under
different cognitive states. Based on extensive simulations, we further confirm the superior performance
of our method on uncovering brain-to-behavior relationships. In the second project, we further
incorporate the disease outcome, and proposed a mediation analysis to explore the effect mechanism
among genetic exposure, structural connectivity and time to disease onset. Extensive simulations
confirmed the superiority of our methods compared with existing alternatives. By applying the methods
to landmark brain connectivity and Alzheimer’s disease studies, we obtained biologically plausible
insights.

</div>

----

<!--

----

<div id="speaker3" style="display: flex; flex-direction: column;">

<h3>Enriching Real-World Data Based Study by Conducting Linkage among Multiple Data Sources; 
[<a href="../docs/slides/AM3-TianyuSun.pdf">Slides</a>], 
[<a href="https://youtu.be/WVOJNiGtV1M" target="_blank">Recording</a>]
</h3>

<h4>Tianyu Sun</h4>

<p>
<img class="imgfloat" src="../docs/Tianyu.jpg"/>
Dr. Tianyu Sun is Manger, Real-World Evidence & Analytics, Biostatistics and Programming at Moderna. He is partnering with cross-functional groups to support the development and execution of Real-World Data (RWD) based studies to enable disease insights and to enhance clinical development. Prior to joining Moderna, Tianyu had experience of different RWD sources, such as Medicare fee-for-service, Optum, RI Medicaid, etc. He also had previously been working as SAS programmer for clinical trial data analysis. Full list of publications: 
<a href="https://scholar.google.com/citations?user=fB8A030AAAAJ&hl=en&oi=sra" target="_blank">link</a>; 
ORCID: <a href="https://orcid.org/0000-0002-3996-1381" target="_blank">link</a>.
</p>


<h4>Abstract</h4>

The real-world data (RWD), such as claims and electronic health records (EHRs), has been widely used for observational studies to generate real-world evidence (RWE) of effectiveness and safety profile for treatments and interventions. However, each data source by themselves has limitations, even data collected by clinical trials. For instance, administrative claims data usually lacks the granularity of lab test results, meanwhile clinical trials are sometime been criticized for a short follow-up period to observe long-term outcomes or adverse events. Conducting linkage (tokenization, when deterministic linkage is not feasible) across different data sources would be able to enrich data elements and to extend the longitudinal data with minimum interference with study subjects. There are several published studies provided insights into linking different data sources and assessing the validity of outcomes obtained from RWD. The results open a door to enrich a single source data and to judicate endpoints collected by clinical trial by conducting linkage across different databases. Furthermore, it demonstrated a chance of extending clinical trial follow-up by linking with RWD for certain endpoints.

</div>



----

<div id="speaker4" style="display: flex; flex-direction: column;">

<h3>Improving Prediction of Adolescent Suicide Attempts with Electronic Health Records by Leveraging External Data Sources; 
[<a href="../docs/slides/AM4-ShaneSacco.pdf">Slides</a>], 
[<a href="https://youtu.be/CAwX3cA3zkg" target="_blank">Recording</a>]
</h3>

<h4>Shane Sacco</h4>

<p>
<img class="imgfloat" src="../docs/shane.jpg"/>

Dr. Shane J Sacco is a Postdoctoral Research Associate in the Department of Statistics at the University of Connecticut and a member at the Center for Population Health at the University of Connecticut Health Center. He received his PhD in Health Promotion Science at the University of Connecticut, with his dissertation focusing upon applied methods to improve nonlinear interaction testing in the social sciences. His current research aims to improve suicide risk prediction using transfer-learning techniques, clinical decision-making pipelines, and feature and outcome engineering.

</p>


<h4>Abstract</h4>

Suicide among adolescents and young adults has risen in the US over the last decade and has driven mandates for risk identification methods such as risk algorithms using electronic health records. However, published risk algorithms heretofore only identify around half of attempts on average with 90% specificity. With this, efforts to improve algorithms are still critical. One potential method to improve existing algorithms is to transfer features important to risk that are not found in analyzed electronic health records by means of data fusion, a principle within transfer learning. In the present talk, I will discuss an established framework to generate additional features for prediction by matching patients with those found in external sources, as well as, case studies to exhibit how and what additional information may be leveraged to improve prediction of suicide attempts in adolescent populations.

</div>


----

<div id="speaker5" style="display: flex; flex-direction: column;">

<h3>Indirect Treatment Comparisons: When Are They Needed and How Do They Work? 
[<a href="../docs/slides/PM1-HaitaoChu.pdf">Slides</a>], 
[<a href="https://youtu.be/y1EzinNn8CY" target="_blank">Recording</a>]
</h3>

<h4>Haitao Chu</h4>

<p>
<img class="imgfloat" src="../docs/HaitaoChu.jfif"/>
Dr. Haitao Chu is a Senior Director in the Statistical Research and Data Science Center at Pfizer Inc. Before he joined Pfizer in April 2022, he was a tenured full Professor of Biostatistics in the University of Minnesota Twin Cities since 2017. He has published over 240 articles and is a Fellow of American Statistical Association. He currently serves as an Associate Editor for the Journal of the American Statistical Association and the American Journal of Epidemiology.
</p>


<h4>Abstract</h4>

In many markets, gaining approval by national reimbursement, pricing, or health technology assessment agencies is critical to the commercial success of a new product launch. In the absence of direct evidence from head-to-head RCTs, private and national payers often expect to see evidence from indirect treatment comparisons including Bucher’s method, network meta-analysis (NMA) or population adjusted indirect comparison (PAIC) to demonstrate the clinical value of the new technology. In this talk, I will provide an overview on indirect treatment comparisons focusing on when are they needed and how do they work? In addition, we will discuss the assumptions, advantages, and disadvantages of each approach. In practice, it is important to consider multiple approaches as sensitivity analyses and to provide totality of evidence as indirect treatment comparison is at high risk of bias no matter which approach is chosen. 

</div>

----

<div id="speaker6" style="display: flex; flex-direction: column;">

<h3>Disentangling the Clinical Complexity of Heterogeneous Diseases through Subphenotyping on Large Scale Electronic Health Records with Machine Learning; 
[<a href="../docs/slides/PM2-FeiWang.pdf">Slides</a>], 
[<a href="https://youtu.be/FkospzWfCrs" target="_blank">Recording</a>]
</h3>

<h4>Fei Wang</h4>

<p>
<img class="imgfloat" src="../docs/FeiWang.jpeg"/>
Fei Wang is currently an Associate Professor of Health Informatics in Department of Population Health Sciences, Weill Cornell Medicine, Cornell University. His major research interest is machine learning and its applications in health data science. He has published more than 300 papers in AI and medicine, which have received more than 23K citations Google Scholar. His H-index is 73. His papers have won 8 best paper awards at top international conferences on data mining and medical informatics. His team won the championship of the PTHrP results prediction challenge organized by the American Association of Clinical Chemistry in 2022, NIPS/Kaggle Challenge on Classification of Clinically Actionable Genetic Mutations in 2017 and Parkinson’s Progression Markers’ Initiative data challenge organized by Michael J. Fox Foundation in 2016. Dr. Wang is the recipient of the NSF CAREER Award in 2018, the inaugural research leadership award in IEEE International Conference on Health Informatics (ICHI) 2019. Dr. Wang is a Fellow of the American Medical Informatics Association (AMIA), a Fellow of the International Academy of Health Sciences and Informatics (IAHSI), a Fellow of the American College of Medical Informatics (ACMI), and a Distinguished Member of the Association for Computing Machinery (ACM). 
</p>

<h4>Abstract</h4>

Diseases such as COVID-19 are highly complex in patients’ clinical manifestations. Disentanglement of such clinical complexity into subpehnotypes can reveal the hidden patterns of the disease and inform stratified medicine. Electronic Health Records (EHRs) are important sources containing rich clinical information but there are many inherent challenges for analyzing them, such as sparsity, high-dimensionality and temporality. In this talk, taking COVID-19 as an example, I will introduce our research on deriving disease subphenotypes from large scale EHR repositories using machine learning approaches, point out the challenges and future research directions. 

</div>


----

<div id="speaker7" style="display: flex; flex-direction: column;">

<h3>Meta-regression Case Studies in Infectious Disease Epidemiology; 
[<a href="../docs/slides/PM3-JoshuaWarren.pdf">Slides</a>], 
[<a href="https://youtu.be/4k1Aw3ApRNU" target="_blank">Recording</a>]
</h3>

<h4>Joshua L. Warren</h4>

<p>
<img class="imgfloat" src="../docs/joshua.jpeg"/>
Joshua Warren is an Associate Professor in the Department of Biostatistics at the Yale School of Public Health. He received his Ph.D. in statistics from North Carolina State University in 2011. Dr. Warren’s research focuses on statistical methods in public health with an emphasis on environmental health problems. Much of his work involves introducing spatial and spatiotemporal models in the Bayesian setting to learn more about associations between environmental exposures, such as air pollution, and various health outcomes including preterm birth, low birth weight, and congenital anomalies. He also has interest in developing and applying spatiotemporal models in collaborative settings such as epidemiology, geography, nutrition, and glaucoma research. His theoretical and methodological interests include multiple topics in spatial/spatiotemporal modeling and Bayesian nonparameterics.
</p>

<h4>Abstract</h4>

Meta-analyses and meta-regressions are becomingly increasingly common in infectious disease epidemiology for interpolating important measures across different spatial locations and time points. However, the standard methods of analysis often need to be extended to accommodate unique features of the collected data and to more directly answer primary hypotheses posed by the research group. In this talk, I discuss two recent examples where new statistical methodology within the hierarchical Bayesian framework was developed to (i) estimate the burden of tuberculosis in incarcerated persons and (ii) quantify the efficacy and effectiveness of a vaccine.      

</div>


-->